build.gradle

```diff
@@ -0,0 +1,106 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:4.0.0'
+  }
+}
+
+plugins {
+  id "com.diffplug.gradle.spotless" version "3.14.0"
+}
+
+ext {
+  osgi_version = "5.0.0"
```

Comment:
✏️ I'm going to recommend using [`buildSrc`](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources) to consolidate your version information, library definitions, etc.

An example of its use can be seen [here, in the SAMLCTK](https://github.com/codice/saml-conformance/tree/master/buildSrc).

Reply:
Yeah I was already planning on moving in that direction since it is way cleaner :+1: